src/pages/status/migration/index.jsx

```diff
@@ -8,13 +8,13 @@ import styles from "./styles.module.css";
 
 // { Done, In PR, Awaiting PR, Awaiting parents, Not solvable, Bot error }
 const ORDERED = [
```

Comment:
Can we add a comment about the boolean meaning here?
```suggestion
// The third value is a boolean representing the default display state on load
// 'true' means visible, 'false' means hidden
const ORDERED = [
```